pkg/cli-runtime/printer/table/tablegenerator.go

```diff
@@ -154,7 +154,9 @@ func (h *HumanReadablePrinter) TableHandler(columnDefinitions []metav1beta1.Tabl
 // ValidateRowPrintHandlerFunc validates print handler signature.
 // printFunc is the function that will be called to print an object.
 // It must be of the following type:
-//  func printFunc(object ObjectType, options PrintOptions) ([]metav1beta1.TableRow, error)
+//
+//	func printFunc(object ObjectType, options PrintOptions) ([]metav1beta1.TableRow, error)
```

Comment:
Remove changes in this file. It is not related to the PR

Reply:
Done